tools/pyorac/arguments.py

```diff
@@ -416,7 +418,7 @@ def check_args_cc4cl(args):
     from pyorac.local_defaults import log_dir, extra_lines, retrieval_settings
 
     # Add extra lines files
-    extra_lines.update({key + '_extra' : val for key, val in args.extra_lines})
+    extra_lines.update({key + '_extra': val for key, val in args.extra_lines})
```

Comment:
I have already done a lot of this line respacing (last time I tried out PyCharm it did a lot of it automatically) but it's combined with changing the case on the `local_defaults` variables. I'll see if I can merge this with that.

Reply:
Yep, it's all automatic - i just checked afterwards to make sure it wasn't being stupid.